Datasets/data-connect-dataset-teamschannelactivity.md

```diff
@@ -0,0 +1,94 @@
+
+## Scenarios
+
+The following are business scenarios that you can answer with this dataset:
+
+- Analyze Microsoft Teams Channels created or deleted by users in a particular time range 
+
+## Questions
+
+The following are examples of questions that you can answer with this dataset:
+
+- How many channels were interacted with by users during a time range? 
+- What is the average number of people leaving or joining a channel? 
+- How often are the same channels used for different purposes (i.e., the channel is renamed)? 
+
+## Joining with other datasets
+
+- The TeamsChannelActivity_v0 dataset can be joined with other activities datasets, to get insights from how employees in the organization interact with and through outlook and teams 
+- It can also be used with TeamsChannelDetails_v0 to get more information regarding the channels queried here.
+
+## Definitions
+
+- **Signal:**  A Substrate signal is a high-value event that captures a user's behavior. A signal can be either active (like joining a meeting) or passive (like a location change).
+
+
+## Schema
+
+| Name  | Type  |  Description  |  FilterOptions  |  FilterType  |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| SignalType | string | Name of the signal. | No | None |
+| Id | string | The unique identifier of the Signal. | No | None |
+| StartTime | datetime | The date and time when action happens. | No | None |
+| CreationTime | datetime | The date and time when the signal was created. | Yes | Date |
+| EndTime | datetime | The date and time when action ends. If a signal is a point-in-time action, it is same as StartTime. If the action lasts a time period, this is the action end time captured on client. | No | None|
+| ItemType | string | The type of item being acted on. | No | None |
+| AppName | string | The end user client name being used to generate the signal, E.g. "Unknown", "OutlookMobile", "OWA", "OutlookDesktop", etc. | No | None |
+| AadTenantId | string | AadTenantId of the actor (signal generator). | No | None |
+| ThreadName | string | The Channel’s Name. Among the properties defined by the signal producer. | No | None |                
+| PriorThreadName | string | Older Channel’s Name (in case the signal was for renaming the channel). Among the properties defined by the signal producer. | No |  None |               
+| TeamGroupId | string | Unique GroupId of the teams channel. Among the properties defined by the signal producer. | No | None |                
+| TeamName | string | The team name owning the Channel. Among the properties defined by the signal producer. | No | None |             
+| MembersAdded | string | A list of members added (in case the signal was AddToChannel or Join Channel). Among the properties defined by the signal producer. | No | None |               
```

Comment:
I see the datatype of MembersAdded, MembersRemoved,  ThreadMembers are array in schema json as well as sample, can you please update fields with complex datatypes instead of string - in both Schema table as well as json representation?